django_keycloak/errors.py

```diff
@@ -1,3 +1,9 @@
+"""
+Module containing custom errors.
+"""
+import django_keycloak.config as config
```

Comment:
This could be imported as `from django_keycloak.conf import settings`

Reply:
I think this was the solution to avoid the circular imports error. You are safe when importing an entire module. I will test it again